.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
- [ ] Let's remove unnecessary files from your project directly. You can add it to gitignore.